docs/components/alert/index.html

```diff
@@ -1,36 +1,188 @@
+          <li>
+            <h3>Getting Started</h3>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/" class="sidebar-item-link"> Introduction </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/typography/" class="sidebar-item-link">
+              Typography
+            </a>
+          </li>
+          <li class="margin-top-3">
+            <h3>Components</h3>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/alert/" class="sidebar-item-link active">
+              Alert
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/avatar/" class="sidebar-item-link">
+              Avatar
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/badge/" class="sidebar-item-link">
+              Badge
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/button/" class="sidebar-item-link">
+              Button
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/card/" class="sidebar-item-link">
+              Card
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/image/" class="sidebar-item-link">
+              Image
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/input/" class="sidebar-item-link">
+              Input
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/lists/" class="sidebar-item-link">
+              Lists
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/navbar/" class="sidebar-item-link">
+              Navbar
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/modal/" class="sidebar-item-link">
+              Modal
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/rating/" class="sidebar-item-link">
+              Rating
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/toast/" class="sidebar-item-link">
+              Toast
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/grid/" class="sidebar-item-link">
+              Grid
+            </a>
+          </li>
+          <li class="sidebar-item">
+            <a href="/docs/components/slider/" class="sidebar-item-link">
+              Slider
+            </a>
+          </li>
+        </ul>
+      </aside>
+
+      <div class="component-section padding-horizontal-3">
+        <div class="component-section-background"></div>
```

Comment:
What's the use of this div? If its not used please remove it

Reply:
This is the background of the content section with slighly blur effect